test/distributed/_composable/fsdp/test_fully_shard_comm.py

```diff
@@ -573,5 +577,89 @@ def post_backward_with_record(self, *args, **kwargs):
         return post_backward_with_record
 
 
+class TestFullyShardUnshard(FSDPTest):
+    @property
+    def world_size(self) -> int:
+        return min(torch.cuda.device_count(), 2)
+
+    @skip_if_lt_x_gpu(2)
+    @wrapSwapTensorsTest(True)
```

Comment:
when shoudl we use `@wrapSwapTensorsTest(True)` ?

Reply:
Good catch! We do not need this anymore since a PR has landed in core now that defaults to use `swap_tensors` for wrapper subclasses like `DTensor`. This is a remnant from before and should be removed.